Makefile

```diff
@@ -4,6 +4,9 @@ PREFIX?=$(shell pwd)
 # Used to populate version variable in main package.
 VERSION=$(shell git describe --match 'v[0-9]*' --dirty='.m' --always)
 GO_LDFLAGS=-ldflags "-X `go list ./version`.Version $(VERSION)"
+ifeq (${REGISTRY_DEBUG},true)
+	GO_GCFLAGS=-gcflags "-N -l"
```

Comment:
Let's call this `DISABLE_GC_OPTIMIZATION` or something like that. When this is enabled, something like `-noopts` should be added to the version string.